### PR TITLE
Added ElasticSearch operate-index for use multiple zebbe-operate in o…

### PR DESCRIPTION
**Pull request by smartmanru**
…ne elasticsearch instance

### Checklist

Please make sure to follow our [Contributing Guide](../blob/main/CONTRIBUTING.md).

**Before opening the PR:**

- [x] There is no other open [pull request](../pulls) for the same update/change.
- [x] The commits follow our [Commit Guidelines](../blob/main/CONTRIBUTING.md#commit-guidelines).
- [x] Tests for charts are added (if needed).
- [x] The main Helm chart and sub-chart are updated (if needed).
- [x] In-repo [documentation](../blob/main/CONTRIBUTING.md#documentation) are updated (if needed).

**After opening the PR:**

- [x] Did you sign our CLA (Contributor License Agreement)? It will show once you open the PR.
- [x] Did all checks/tests pass in the PR?

### CODE REVIEW

**Comment by aabouzaid** [comment]
@smartmanru please run `GOMAXPROCS=2 make go.test-golden-updated` to update the golden files.

File: charts/camunda-platform/values.yaml [hunk 1]
```diff
@@ -128,6 +128,9 @@ global:
 
       # Identity.auth.operate configuration to configure Operate authentication specifics on global level, which can be accessed by other sub-charts
       operate:
+        #Use another index-prefix for operate
+        elasticsearch:
```
Comment (aabouzaid) [inline charts/camunda-platform/values.yaml:132]: This is the wrong path.
It should be under `operate` not `global.identity.auth.operate`.

Comment (aabouzaid) [reply charts/camunda-platform/values.yaml:132]: @smartmanru this change is in the Helm chart value config, not in the app config. This is not the Operate configuration.
the values here are referenced in the Operate ConfigMap which has the actual Operate config.

Try the following in your branch to see it:
```
helm template charts/camunda-platform
```

**Comment by jessesimpson36** [comment]
This has been open for too long, and I'm still not clear what this PR is doing.  Lets close this PR and discuss in an issue if we think this topic needs to be pushed forward.

**Pull request closed without merging**